AGPs/AGP-Comprehensive-Aragon-Security-Review-Program.md

```diff
@@ -0,0 +1,39 @@
+
+Speaking of devs building on Aragon, the Aragon community is now shipping more projects through A1, Autark, Aragon Black, and the Nest program. These apps are amazing and could unlock tons of value, but only if people use them…
+
+People are wary of DAOs because of “the DAO” hack. For people to trust (and thus use) Aragon we need to go above and beyond to prove that the Aragon platform, and all major releases of Aragon apps, are secure. This is easier said than done.
+- Security audits are not perfect. Even with an audit all you know is what was reported. They might have missed something.
+- Security audits are highly technical and the process is opaque to people who are not involved in the Ethereum security community.
+- Security audits are expensive. Having strategic and financial help to navigate that negotiation is extremely important!
+
+Aragon is trying to attract talent and users. Having the world's easiest to build on and most secure platform is a huge selling point. If developers see that they will have help shipping professional and production ready applications they are more likely to choose to build on Aragon vs other platforms. If users trust that all major Aragon apps are secure, they’re more likely to use them. To do this we need a multi layered approach to security. This can include audits for all major projects (Nest and Flock), but should also include a comprehensive Bug Bounty program that covers all major apps.
+
+The Aragon Association is spending millions of dollars on Flock teams and hundreds of thousands of dollars on Nest projects. Security audits help to protect that investment. Aragon DAOs currently hold hundreds of thousands of dollars in their vaults, and soon more organizations will be using Aragon DAOs to manage millions. It is essential that projects funded via the Aragon Association treasury funds are secure, otherwise no one will use them.
+
+Augmenting the Aragon Security Review program will allow developers to focus on building their applications. It will also allow the Aragon Association to build relationships with trusted security partners in order reducing time and money spent on security audits overall. This is essential for Aragon to be able to provide professional and trusted solutions for organizations on a global scale.
```

Comment:
```suggestion
Augmenting the Aragon Security Review program will allow developers to focus on building their applications. It will also allow the Aragon Association to build relationships with trusted security partners in order to reduce time and money spent on security audits overall. This is essential for Aragon to be able to provide professional and trusted solutions for organizations on a global scale.
```